package.json

```diff
@@ -13,10 +13,11 @@
   "homepage": ".",
   "repository": "https://github.com/brown-ccv/honeycomb",
   "dependencies": {
-    "@brown-ccv/behavioral-task-trials": "^1.21.2",
+    "@brown-ccv/behavioral-task-trials": "^1.24.0",
```

Comment:
there shouldn't be a `package-lock.json` commited to the repo anymore - can you make sure it's in the gitignore on this branch and delete the one that's commited from being tracked?